cds/modules/records/links.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2017 CERN.
+#
+# Invenio is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+"""Links for record serialization."""
+
+from __future__ import absolute_import, print_function
+
+from invenio_records_rest.links import default_links_factory
+
+from .resolver import record_resolver
+from ..deposit.api import deposit_project_resolver
+
+
+def _fill_video_extra_links(record, links):
+    """Add extra links if it's a video."""
+    if '_project_id' in record:
+        # if it's a video, get the deposit project
+        deposit_project = deposit_project_resolver(record['_project_id'])
+        if deposit_project.is_published():
+            # if the project is published, get the record project
+            pid_project, record_project = deposit_project.fetch_published()
+            # include project link, title
+            links['project'] = {
+                'links': default_links_factory(pid_project),
+                'title': record_project.get('title', '')
```

Comment:
This is an information that we need to avoid another request to the server.
Maybe we could go for 
```json
{
  "project": { "href": "....", "title": "..."}
}
```
see e.g. http://stateless.co/hal_specification.html